### PR TITLE
feat: load data for simulation from CSV

### PR DESCRIPTION
**Pull request by falox**
Added the ability to load the scenario to be simulated from a CSV file

### CODE REVIEW

File: cmd/simulate/simulate.go [hunk 1]
```diff
@@ -25,20 +29,22 @@ func must(err error) {
 }
 
 var outputFile = "cluster-health-analyzer-openmetrics.txt"
+var csvFile string
```
Comment (iNecas) [inline cmd/simulate/simulate.go:32]: `scenarioFile` might be better name

File: cmd/simulate/simulate.go [hunk 2]
```diff
@@ -252,9 +258,94 @@ func RelativeToAbsoluteIntervals(relIntervals []utils.RelativeInterval, end mode
...
+
+	return parseIntervalsFromCSV(file)
+}
+
+func parseIntervalsFromCSV(file io.Reader) ([]utils.RelativeInterval, error) {
+	var intervals []utils.RelativeInterval
+	scanner := bufio.NewScanner(file)
+
+	// Skip the first line with header
+	scanner.Scan()
```
Comment (iNecas) [inline cmd/simulate/simulate.go:277]: Why can't we use the standard approach of creating csvReader at the beginning and calling r.Read() for each line? https://go.dev/play/p/XxthE8qqZtZ, we can still count the number of lines and decide to ignore the first one.

**Comment by iNecas** [comment]
Nice addition that will definitely be useful. Some small comments inside.

File: cmd/simulate/simulate.go (hunk 2 above)
Comment (falox) [reply cmd/simulate/simulate.go]: right, simplified

File: cmd/simulate/simulate.go (hunk 1 above)
Comment (falox) [reply cmd/simulate/simulate.go]: absolutely better, renamed

**Pull request merged**